politeiawww/config.go

```diff
@@ -731,9 +732,44 @@ func loadConfig() (*config.Config, []string, error) {
 			}
 		}
 
+	case userDBMysql:
+		// Mysql requires database host setting.
+		if cfg.DBHost == "" {
+			return nil, nil, fmt.Errorf("dbhost param is required")
+		}
+
+		// Validate user database host.
+		_, err = url.Parse(cfg.DBHost)
+		if err != nil {
+			return nil, nil, fmt.Errorf("parse dbhost: %v", err)
+		}
+
+		// Validate user database encryption keys.
+		cfg.EncryptionKey = util.CleanAndExpandPath(cfg.EncryptionKey)
```

Comment:
The encryption key validation should use the same code path for cockroachdb and mysql.

Reply:
Good point, maybe also the host